### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by denisebitca** [denisebitca/blog_posts#3]
A number of accessibility features are completely missing:

- Proper tab support
- Multi-language template support on blog posts that are not in English
- Soft font enforcement (to allow system settings to prevail)

There are probably more accessibility features missing. Would be very important to work on. I'd like to hear more opinions on the missing accessibility features!
